tests/integration/test_plugins.py

```diff
@@ -389,6 +389,22 @@ async def test_docker_build_publish_plugin(unit_web_client: UnitWebClient):
     ), f"docker-build-publish configuration option not found. {config_page}"
 
 
+async def test_reverse_proxy_plugin(unit_web_client: UnitWebClient):
```

Comment:
According to our contribution guide, the code blocks corresponding to the arrange, assert and act should be separated by a blank line. See https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-structure